pkg/build/controller/strategy/custom.go

```diff
@@ -22,12 +23,22 @@ type CustomBuildStrategy struct {
 
 // CreateBuildPod creates the pod to be used for the Custom build
 func (bs *CustomBuildStrategy) CreateBuildPod(build *buildapi.Build) (*kapi.Pod, error) {
-	data, err := bs.Codec.Encode(build)
```

Comment:
It needs to be coming from the storage version config flag.

On Fri, Jan 22, 2016 at 1:51 PM, Martin Nagy <notifications@github.com>
wrote:

> In pkg/build/controller/strategy/custom.go
> <https://github.com/openshift/origin/pull/6760#discussion_r50574745>:
>
> > @@ -22,12 +23,22 @@ type CustomBuildStrategy struct {
> >
> >  // CreateBuildPod creates the pod to be used for the Custom build
> >  func (bs *CustomBuildStrategy) CreateBuildPod(build *buildapi.Build) (*kapi.Pod, error) {
> > -	data, err := bs.Codec.Encode(build)
>
> Yes, all strategies have their codecs set in run_components.go:
>
> https://github.com/openshift/origin/blob/master/pkg/cmd/server/origin/run_components.go#L190-L191
>
> https://github.com/openshift/origin/blob/master/pkg/cmd/server/origin/run_components.go#L203-L218
> Should we just use latest?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6760/files#r50574745>.
>